h/static/scripts/directive/annotation.js

```diff
@@ -598,41 +609,50 @@ function AnnotationController(
     * @description Saves any edits and returns to the viewer.
     */
   vm.save = function() {
-    if (!model.user) {
+    if (!domainModel.user) {
       return flash.info('Please sign in to save your annotations.');
     }
 
-    if (!validate(vm.annotation)) {
-      return flash.info('Please add text or a tag before publishing.');
-    }
-
     // Update stored tags with the new tags of this annotation.
-    var newTags = vm.annotation.tags.filter(function(tag) {
-      var tags = model.tags || [];
+    var newTags = vm.form.tags.filter(function(tag) {
+      var tags = domainModel.tags || [];
       return tags.indexOf(tag.text) === -1;
     });
     tags.store(newTags);
 
     switch (vm.action) {
       case 'create':
-        updateDomainModel(model, vm.annotation);
+        updateDomainModel(domainModel, vm, permissions, groups);
```

Comment:
Indeed, and `updateViewModel()` was not being called in that case, so the live updates were not working. Should be fixed now.